file_paths.md

```diff
@@ -163,160 +163,195 @@ The package will create the following services `puppet`, `mcollective`,
 and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
-    Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    Path                                          Setting                        3.x
+    C:\ProgramData\PuppetLabs *                                                  n/a
```

Comment:
This was always `C:\ProgramData\PuppetLabs`. It's used for storing settings, not the installed application.